yb-voyager/src/dbzm/yb_cdc_client.go

```diff
@@ -154,11 +156,39 @@ func (ybc *YugabyteDBCDCClient) ListMastersNodes() (string, error) {
 		return "", fmt.Errorf("running command with args: %s, error: %s", args, err)
 	}
 	//stdout - Master Addresses: <comma_separated_list_addresses>
-	masterAddresses := strings.Trim(strings.Split(stdout, ": ")[1], " \n")
+	masterAddresses := ""
+	stdoutSplits := strings.Split(stdout, ": ")
+	if len(stdoutSplits) > 1 {
+		masterAddresses = strings.Trim(stdoutSplits[1], " \n")
+	}
 	ybc.ybMasterNodes = masterAddresses
 	return masterAddresses, nil
 }
 
+func (ybc *YugabyteDBCDCClient) GetNumOfReplicationStreams() (int, error) {
+	tserverPort := utils.GetEnvAsInt("YB_TSERVER_PORT", 9100) //TODO: make it internally handled by yb-client
+	args := fmt.Sprintf("-get_num_of_cdc_streams -master_addresses %s -tserver_port %d", ybc.ybServers, tserverPort)
+
+	if ybc.sslRootCert != "" {
+		args += fmt.Sprintf(" -ssl_cert_file '%s'", ybc.sslRootCert)
+	}
+
+	stdout, err := ybc.runCommand(args)
+	if err != nil {
+		return 0, fmt.Errorf("running command with args: %s, error: %s", args, err)
+	}
+	//stdout - Streams: <num_of_streams>
+	numOfStreams := 0
+	stdoutSplits := strings.Split(stdout, ": ")
+	if len(stdoutSplits) > 1 {
```

Comment:
wouldn't it be an error if len(stdoutSplits) is not greater than 1? 
Also, let's be specific. We expect no. of splits to be exactly 2, correct? Anything other than that, we should return an error.